sig-multicluster/management-cluster-position-statement.md

```diff
@@ -0,0 +1,46 @@
+# Management Cluster - SIG Multicluster Position Statement
+
+Author: Corentin Debains (**[@corentone](https://github.com/corentone)**), Google  
+Last Edit: 2024/12/09  
+Status: DRAFT
+
+## Goal
+To establish a standard definition for a central cluster that is leveraged by multicluster 
+controllers to manage multicluster applications or features across an inventory of clusters.
+
+## Context
+Multicluster controllers have always needed a place to run. This may happen in external
+proprietary control-planes but for more generic platforms, it has been natural for the
+Kubernetes community to leverage a Kubernetes Cluster and the existing api-machinery
+available. There has been a variety of examples of which we can quote ArgoCD, MultiKueue
+or any of the Federation effort (Karmada, KubeAdmiral), all of them not-naming the "location"
+where they run or not aligning on the name (Admin cluster, Hub Cluster, Manager Cluster...). 
+The [ClusterInventory](https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/4322-cluster-inventory/README.md)
+(ClusterProfile CRDs) is also the starting point for a lot of multicluster controllers and, 
+being a CRD, it requires an api-machinery to host it.
+
+## Definition
+
+A (multicluster) management cluster is a Kubernetes cluster that acts as a 
+control-plane for other Kubernetes clusters (named Workload Clusters to differentiate
+them). It MUST have visibility over the available clusters and MAY have administrative
```

Comment:
I wonder what does "visibility" mean here? There can be workload clusters that do not have public IP.

Reply:
Agreed, visibility is vague. I think it is somewhat the ability to query more data about the cluster. I'm thinking it would be the ability to "READ" via kubectl (thought it may not be everything). But it should be able to potentially see the Cluster object in GKE or other platforms (wherever the provisioning/lifecycle happens).

Maybe this is too vague, I originally had this in to mandate some kind of visibility on the cluster so that the controller could do something (if the controller has only access to the clusterprofile, I dont know what it can really do).

But we may be able to remove it and it wouldn't affect the definition much.